utils/v2tofhirr4/Ballerina.toml

```diff
@@ -6,4 +6,4 @@ keywords = ["Healthcare", "HL7", "FHIR"]
 name = "health.hl7v2.utils.v2tofhirr4"
 org = "ballerinax"
 repository = "https://github.com/ballerina-platform/module-ballerinax-health.hl7v2"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
No need to release v2tofhir. Hl7v2 and v2tofhir have already been migrated to bal 5 and released.

Reply:
ok. I thought it was only Hl7v2

Reply:
Removed